doc/lua_api.txt

```diff
@@ -4743,6 +4743,12 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Flags for schematic decorations.  See 'Schematic attributes'.
         rotation = "90" -- rotate schematic 90 degrees on placement
     --  ^ Rotation can be "0", "90", "180", "270", or "random".
+        y_offset = 1,
+    --  ^ Y offset of the base node layer of the schematic relative to the
+    --  ^ 'place_on' node, can be positive or negative.
+    --  ^ If the flag 'place_center_y' is set this parameter is ignored.
+    --  ^ If absent or zero no vertical offset is applied and the schematic
+    --  ^ base node layer is at the level of the 'place_on' node.
```

Comment:
Explain what this means practically. I.e. that the schematic is “buried into” the place_on node by default.

Reply:
It is clear and precise language already, i'm not going to dumb it down :)